app/views/settings/edit.html.erb

```diff
@@ -3,7 +3,7 @@
 <div class="t-body">
   <h2><%= t ".mfa.multifactor_auth" %></h2>
   <% if @user.mfa_enabled? %>
-    <p><%= t ".mfa.enabled" %></p>
+    <p><%= t(".mfa.enabled").html_safe %></p>
```

Comment:
per 

https://api.rubyonrails.org/v6.0.3.2/classes/ActionView/Helpers/TranslationHelper.html#method-i-translate

You can rename that key to `.mfa.enabled_html` and it will be `html_safe` by default.